packages/nextjs/app/builders/0x423775bd841f9d3c3120c103A14a64EBc7B4F40A/page.tsx

```diff
@@ -0,0 +1,61 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import { ChatBubbleLeftRightIcon, CodeBracketIcon, PaperAirplaneIcon } from "@heroicons/react/24/outline";
+
+const SocialButton: React.FC<{ title: string; href: string; children: React.ReactNode }> = ({
+  title,
+  href,
+  children,
+}) => {
+  return (
+    <a href={href} className="btn btn-primary w-24 xs:w-32 sm:w-36">
+      {children}
+      {title}
+    </a>
+  );
+};
+
+const EvilPage: NextPage = () => {
+  return (
+    <div className="flex flex-col flex-grow justify-center items-center pt-4">
+      <Image
+        className="mask mask-squircle"
+        width={200}
+        height={200}
+        src="/3vil7win-avatar.webp"
+        alt="An alien with a katana, a bitcoin monogram jacket, and a t-shirt that reads 'Believe in Yourself.'"
+      />
+      <div className="hero bg-base-200">
+        <div className="hero-content text-center">
+          <div className="max-w-md">
+            <h1 className="text-4xl font-bold">Hey Fren</h1>
+            <p className="py-1">
+              I&apos;m 3vil7win (ET for short), a creative developer primarily interested in Three.js/WebGL and
+              EVM/Solidity.
+            </p>
+            <p className="py-1">
+              My programming journey began with Lego Dacta and a TI-83 calculator, and I&apos;ve followed web3 off and
+              on since the Mt.Gox and BTC-e trollbox days.
+            </p>
+            <p className="py-1">
+              I&apos;m excited to join BuidlGuidl to work on my chops, meet other builders, and make some cool OSS.
+            </p>
+            <div className="flex justify-center gap-3 pt-3">
+              <SocialButton title="Github" href="https://github.com/3vil7win">
```

Comment:
You might want to consider having links open in a new tab.

Reply:
Is there a way to do this where the user still has the option to not open links in another tab? I prefer to give the user all the options, but I'm not aware of a way to do that while the link is set to open a new tab.

Reply:
On macOS, users can hold the Command key and click on a link to open it in a new tab automatically. This built-in functionality provides users the option to open links in new tabs without requiring additional code. But user have to know it of course :)

Reply:
I can set the target to _blank but then users will no longer have the option to open links in the same tab. It would be nice if the keybind was simply flipped and command+click would open in the same tab, then, but it does not. It's a big pet peeve of mine when sites force me to open new tabs, but I can implement that if you insist.